src/main/webapp/resources/js/types/irida/index.d.ts

```diff
@@ -131,6 +131,7 @@ declare namespace IRIDA {
     analysisPriority: string;
     users: User[];
     analysisTemplates: string[]; // TODO (Josh - 6/7/22): What should this be
+    url: string;
```

Comment:
Do we want to name this `remoteUrl` to directly map to the server Project model?  That is what it is used for right?

Reply:
Ok, I was just doing what was easiest for me in the moment. Please see b9fae817b6f651d44e6e9ada08fd052af4814eef.